domains/zlan.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "zlance88",
+        "email": "zlance88@outlook.com"
+    },
+    "record": {
+        "A": ["35.215.6.101"]
```

Comment:
I cannot reach your website
"This site can’t be reached" (`DNS_PROBE_FINISHED_NXDOMAIN`)

Reply:
The site was designed with another domain name "zleen.cf". But that domain DNS shutdown yesterday.

Reply:
Shall I change it to "zlen.is-a.dev" now , to be approved?

Reply:
> The site was designed with another domain name "zleen.cf". But that domain DNS shutdown yesterday.

I cannot reach `zleen.cf` too with same error

> Shall I change it to "zlen.is-a.dev" now , to be approved?

the `zlen` doesn't exist in the repo

Reply:
sorry I'm confused. 
I'm trying to register a new domain, to be used for my website, which is not running yet. Is it required that site shall be up-and -running before approval?

Reply:
> sorry I'm confused. I'm trying to register a new domain, to be used for my website, which is not running yet. Is it required that site shall be up-and -running before approval?

yes, if you see https://github.com/is-a-dev/register/blob/main/.github/PULL_REQUEST_TEMPLATE.md there's a check "You have completed your website." under the requirements